source/deployment/advanced-permissions.rst

```diff
@@ -0,0 +1,191 @@
+
+Post Management
+~~~~~~~~~~~~~~~~
+
+**Restrict who can delete posts**
+
+Example: As the default for the entire system, restrict deleting posts to only Team and System Admins.
+
+1. In **System Console** > **Advanced Permissions**, edit the **System Scheme**
+2. In the **All Members** and **Channel Admin** panels, uncheck the boxes for **Delete Own Posts** and **Delete Others Posts**
+3. In the **Team Admin** and **Channel Admin** panels, check the box boxes for **Delete Own Posts** and **Delete Others Posts**
+
+**Restict who can edit posts**
+
+Example: As the default for the entire system, only allow users to edit their own posts for five minutes after posting.
+
+1. In **System Console** > **Advanced Permissions**, edit the **System Scheme**
+2. In the **All Members**, **Channel Admin** and **Team Admin** panels, check the box for **Edit Posts**
+3. From any panel, click the gear button to set the global time limit to ``300`` seconds.
+
+.. note::
+
+  The post edit time limit is a `global config variable <https://docs.mattermost.com/administration/config-settings.html#post-edit-time-limit>`_ ``PostEditTimeLimit``, so setting a post edit time limit applies system-wide to all teams and roles.
+
+
+Integration Management
+~~~~~~~~~~~~~~~~~~~~~~~
+
+**Restict managing webhooks and slash commands**
+
+Example: As the default for the entire system, only allow System Admins to create, edit and delete integrations.
+
+1. In **System Console** > **Advanced Permissions**, edit the **System Scheme**
+2. In the **All Members**, **Channel Admin** and **Team Admin** panels, uncheck the boxes for **Manage Webhooks** and **Manage Slash Commands**.
+
+.. note::
+
+  Permissions for creating, editing and deleting integrations are currently grouped for each integration type. These will be split into separate permissions in a future release.
+
+Administration Tools
+--------------------
+
+There are a number of CLI tools available for Admins to help in configuring and troubleshooting the permissions system:
+
+1. `Reset to default permissions <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-permissions-reset>`_: Resets all permissions to the default on new installs.
+2. `Export permission schemes <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-permissions-export>`_: Exports the System Scheme and any Team Override Schemes to a jsonl file.
+3. `Import permission schemes <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-permissions-import>`_: Imports the System Scheme and any Team Override Schemes to your Mattermost instance from a jsonl input file in the format outputted by ``mattermost permissions export``.
+
+Backend Infrastructure
+-----------------------
+
+Technical Admins or developers looking for a deeper understanding of the permissions backend can refer to our :doc:`permissions-backend` technical documentation.
+
+Glossary
```

Comment:
This is awesome! 0/5, perhaps we mention there's a glossary somewhere in the intro? That way a reader knows it exists.

(I know it's in the TOC but at least I missed it)